azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -1335,6 +1445,45 @@ func flattenKubernetesClusterNetworkProfile(profile *containerservice.NetworkPro
 		values["load_balancer_sku"] = string(profile.LoadBalancerSku)
 	}
 
+	lbProfiles := make([]interface{}, 0)
+
+	if profile.LoadBalancerProfile != nil {
+		lb := make(map[string]interface{})
+
+		if profile.LoadBalancerProfile.ManagedOutboundIPs != nil {
+			if profile.LoadBalancerProfile.ManagedOutboundIPs.Count != nil {
+				lb["managed_outbound_ips"] = profile.LoadBalancerProfile.ManagedOutboundIPs.Count
+			}
+		}
+
+		if profile.LoadBalancerProfile.OutboundIPs != nil {
+			if profile.LoadBalancerProfile.OutboundIPs.PublicIPs != nil {
+				lb["outbound_ip_address_ids"] = profile.LoadBalancerProfile.OutboundIPs.PublicIPs
+			}
+		}
+
+		if profile.LoadBalancerProfile.OutboundIPPrefixes != nil {
+			if profile.LoadBalancerProfile.OutboundIPPrefixes.PublicIPPrefixes != nil {
+				lb["outbound_ip_prefixes_ids"] = profile.LoadBalancerProfile.OutboundIPPrefixes.PublicIPPrefixes
```

Comment:
taking a closer look into this since this returns a `*[]ResourceReference` we'll need to flatten this to a `[]string` so that this sets correctly - I hope you don't mind but I'm going to do this as a part of the rebase